src/main/java/uk/gov/hmcts/reform/coh/service/QuestionRoundService.java

```diff
@@ -33,23 +33,36 @@ public boolean isQrValidState(Question question, OnlineHearing onlineHearing) {
         int targetQuestionRound = question.getQuestionRound();
         int currentRoundNumber = getCurrentQuestionRoundNumber(onlineHearing);
 
-        Optional<QuestionState> optionalQuestionState = questionStateService.retrieveQuestionStateByStateName(ISSUED);
-        if(!optionalQuestionState.isPresent()){
+        Optional<QuestionState> optionalIssuedState = questionStateService.retrieveQuestionStateByStateName(ISSUED);
+        if(!optionalIssuedState.isPresent()){
```

Comment:
This could be replaced with
```java
QuestionState issuedState = questionStateService.retrieveQuestionStateByStateName(ISSUED)
  .orElseThrow(() -> new NoSuchElementException("Error: Required state not found"));
```